### ISSUE TITLE
PyYAML 5.1 warning

### ISSUE THREAD

**Issue by andypiper** [twitterdev/search-tweets-python#66]
Currently, running searchtweets against the latest PyYAML release results in a warning on loading the configuration file:

```
credentials.py:34: YAMLLoadWarning: calling yaml.load() without Loader=… is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
```

More information in [this issue](https://github.com/yaml/pyyaml/issues/265).

This is a minor / low priority fix, as the syntax is currently only deprecated, and the code will still work despite the warning being issued.

**Comment by jrmontag** [comment]
Thanks for reporting @andypiper.

**Comment by fionapigott** [comment]
Seems fixed by using safe_loader, #68
